.github/workflows/ci.yaml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on:
+  push:
+    branches:
+      - '*'
+    tags-ignore:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    name: Run syntax_tests
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+        with:
+          path: Terraform
+
+      - name: Download syntax_tests
+        run: wget -O st_syntax_tests.tar.xz https://download.sublimetext.com/st_syntax_tests_build_4094_x64.tar.xz
```

Comment:
Is it worth testing multiple versions in a matrix at some point? Not necessarily asking you to make that change, but rather tapping into your knowledge of ST API. 😄

Reply:
Added testing in ST3 also.